spec/metadata.spec.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+describe("METADATA", () => {
+    var artifact, space;
+
+    beforeAll(() => {
+        return jasmine.shelf.createSpace().then((testSpace) => {
+            space = testSpace;
+        });
+    });
+    beforeEach(() => {
+        return space.uploadArtifact("LOL").then((a) => {
+            artifact = a;
+        });
+    });
+    it("can be uploaded and downloaded.", () => {
+        var newMetadata;
+
+        newMetadata = {
+            myNewProperty: {
+                value: "Hi there"
+            }
+        };
+
+        return artifact.metadata.updateAll(newMetadata).then(() => {
+            return artifact.metadata.getAll();
+        }).then((allMetadata) => {
+            expect(allMetadata.myNewProperty.value).toEqual(newMetadata.myNewProperty.value);
+            [
+                "md5Hash",
+                "artifactPath",
+                "artifactName"
+            ].forEach((prop) => {
+                expect(allMetadata[prop]).not.toBe();
```

Comment:
If you don't want to put `null` in the `toBe()` because ESLint complains, you can use `toBeNull()`.